tower-http-service/src/lib.rs

```diff
@@ -33,37 +44,20 @@ pub trait HttpService<RequestBody>: Sealed<RequestBody> {
     /// `HttpService` instance needs to be used where a `T: Service` is
     /// required, it must be wrapped with a type that provides that
     /// implementation. `LiftService` does this.
-    fn lift(self) -> LiftService<Self> where Self: Sized {
-        LiftService { inner: self }
+    fn into_service(self) -> util::IntoService<Self> where Self: Sized {
```

Comment:
why not just import these instead of using the full path?

Reply:
🤷‍♂️